tests/dags/subdir1/test_ignore_this.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from datetime import datetime
+
+from airflow.models import DAG
+from airflow.operators.python_operator import PythonOperator
+
+
+def raise_error():
+    raise Exception("This dag shouldn't have been executed")
```

Comment:
It won't be -- it's never going to be unpaused.

Instead you should just have this file contain this:

```
form airflow.models import DAG  # noqa - needed to work against airflow "safe mode" parsing

raise Exception("This dag shouldn't have been executed")
```

Reply:
Agreed, I'm changing this to the code you gave above.